Mail2Bug/Email/EmailBodyProcessingUtils.cs

```diff
@@ -3,15 +3,21 @@
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
+using CsQuery;
 
 namespace Mail2Bug.Email
 {
     public class EmailBodyProcessingUtils
     {
         public static string GetLastMessageText(IIncomingEmailMessage message)
+        {
+            return message.IsHtmlBody ? GetLastMessageText_Html(message.RawBody) : GetLastMessageText_PlainText(message);
```

Comment:
should the new behavior be opt-in from the config xml? at least until we can vet it in our team?

Reply:
It's now opt-in